safenode/src/network/msg/mod.rs

```diff
@@ -7,8 +7,8 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 mod codec;
+pub use codec::{Cmd, CmdResponse, Query, QueryResponse, Request, Response};
```

Comment:
the naming of `Cmd`, `Query` and `Request/Response` is bit overlapping.
If do want to divide the communications based on the usage, better give a more explicit naming.
and same if divide based on the targets (i.e. client-node, or among nodes)

Reply:
Do you think, `Action / ActionResult` and `Query / QueryResult` be a more apt name? The client should be at a higher level, so should we distinguish between them at this layer?

Reply:
I think currently we can just stay with `Request/Response` or `Action/AcitonResult`, to distinguish the message direcion, shall be enough.